cytotable/utils.py

```diff
@@ -188,10 +189,10 @@ def _sqlite_mixed_type_query_to_parquet(
             A str which is a path to a SQLite database file.
         table_name: str:
             The name of the table being queried.
-        chunk_size: int:
-            Row count to use for chunked output.
-        offset: int:
-            The offset for chunking the data from source.
+        page_key: str:
+            The column name to be used as the key for pagination.
```

Comment:
Might be helpful to define the variable without using the variable name. Perhaps something like "The column name to be used to identify and join different pagination chunks"?

Reply:
Thanks, that's more clear - I'll update this accordingly.